build.gradle.kts

```diff
@@ -202,9 +203,22 @@ fun createDocsIndexPage(): String {
                         div("cover") {
                             h2 { +"Available versions:" }
                             div("table") {
+                                val regex = """(\d*).(\d*).(\d*)""".toRegex()
+                                val comparator = Comparator { a: File, b: File ->
```

Comment:
Use `org.semver.Version.parse()` instead of implementing your own comparator. (is part of `org.semver:api:0.9.33`).

Maybe it's also better to move this HTML generating code into a separate file in the `buildSrc` folder.

Reply:
I agree, it would be nicer to extract the helper methods in a separate file. But I ran into some issues with gradle when trying to do that.